af/fastpath/fastpath/utils.py

```diff
@@ -261,60 +612,95 @@
+        # https://explorer.ooni.org/measurement/20170915T220312Z_AS22935_6FKfune3ZuFavfORPXNul209Ffwv3jL7RyjzMJppxYju2caAoE?input=http:%2F%2Fwww.breastenlargementmagazine.com
         {
             "header_name": "Location",
             "header_prefix": "http://ibossreporter.edutech.org/block/bp.html",
-            "locality": "country",
+            "locality": "local",
         },
+        # https://explorer.ooni.org/measurement/20170628T182856Z_AS25605_Lev8VClbbZplNkYfBGujzPiKFI7rHxERx9SNwOfuR1M8WCTBSZ?input=http:%2F%2Fwww.aceshigh.com
         {
             "header_name": "Location",
             "header_prefix": "http://alert.scansafe.net/alert/process",
-            "locality": "country",
+            "locality": "local",
         },
+        # https://explorer.ooni.io/measurement/20170722T013346Z_AS14340_QXOwhyfxJUPRGWsCqanoOycTnbHcpU4CW3NBNXUMMbxbi3Q6I3?input=http:%2F%2Fwww.blubster.com
         {
             "header_name": "Location",
             "header_prefix": "http://184.168.221.96:6080/php/urlblock.php",
-            "locality": "country",
+            "locality": "local",
         },
         {
             "header_name": "Location",
             "header_prefix": "https://gateway.wifast.com:443/wifidog/login/",
             "locality": "local",
         },
+        # https://explorer.ooni.org/measurement/20190205T191943Z_AS26638_QbFGhgqZ8sqXmCQrZNrNgB0RWB6EUfjFYPbKYOgaihiWLv5xNb?input=http:%2F%2Ftwitter.com%2F
         {
             "header_name": "Location",
             "header_prefix": "https://mpswebfilterwashbu.mpls.k12.mn.us:6082/php/uid.php",
             "locality": "local",
         },
+        # https://explorer.ooni.org/measurement/20180406T152727Z_AS39942_lquPnt0vjeXydfleOdSkxyjst6VTiUWb58f3x5qlFKTSlTIQLG?input=http:%2F%2Fflirtylingerie.com%2F
+        # vendor: forcepoint
+        {
+            "body_match": "<title>Access to this site is blocked</title>",
+            "locality": "local"
+        },
+        # https://explorer.ooni.org/measurement/20171129T155619Z_AS11714_vJUMktHjy0cQGKqYqY3fgOQQLVNfnxb1V11fvP6jTXTbbTX60e?input=http:%2F%2Fwww.pandora.com
+        # vendor: netsweeper
+        {
+            "body_match": "It is a good idea to check to see if the NetSweeper restriction is coming from the cache of your web browser",
+            "locality": "local"
+        },
     ],
     "VN": [
+        # https://github.com/ooni/pipeline/issues/186
+        # https://explorer.ooni.org/measurement/20130506T043500Z_IBzthJbAAPsdFZQLPrjMbAwlELzGqtabqMatKpBqqBSWnUyQnA?input=http:%2F%2Fwww.sos-reporters.net
         {
             "header_name": "Location",
             "header_prefix": "http://ezxcess.antlabs.com/login/index.ant?url",
             "locality": "local",
         }
     ],
     "ZZ": [
-        {
-            "header_full": "Kerio Control Embedded Web Server",
```

Comment:
I think that applies to many other case too though. I put in the real country of the measurement because I assumed you had automatically put it in ZZ because the probe_cc in the measurement was set to `ZZ`, yet the `resolver_address` mapped to a CL network.